lib/src/widgets/board.dart

```diff
@@ -557,6 +557,10 @@ class _BoardState extends State<Board> {
         _premoveDests = null;
       });
     }
+
+    if (widget.settings.inputMethod == InputMethod.drag) {
```

Comment:
This follows the Web UI, which is good.

I think we should write a test to verify that in drag only mode the piece is automatically unselected on tap up.

Reply:
Hmm isn't that what `Cannot move by tap if piece shift method is drag` tests? This test taps e2 (i.e. pointer down then up) and checks that this does not select anything